src/components/big-interactive-pages/editor.tsx

```diff
@@ -71,13 +72,36 @@ const saveGame = debounce(800, (persistenceState: Signal<PersistenceState>, code
 	lastSavePromise = (lastSavePromise ?? Promise.resolve()).then(doSave)
 })
 
+const exitTutorial = (persistenceState: Signal<PersistenceState>) => {
+	if (persistenceState.value.kind === 'PERSISTED') {
+		delete persistenceState.value.tutorial
+		if (typeof persistenceState.value.game !== 'string') {
+			delete persistenceState.value.game.tutorialName
+		}
+		persistenceState.value = {
+			...persistenceState.value,
+			stale: true
+		}
+		if (persistenceState.value.kind === 'PERSISTED') {
+			persistenceState.value = {
+				...persistenceState.value,
+				cloudSaveState: 'SAVING'
+			}
```

Comment:
Is this check doing anything and do you need to set `persistenceState.value` twice?

Reply:
You can only set the `cloudSaveState` in `PERSISTED` games from what I've noticed.

Reply:
Yeah, but aren't you checking that already in the outer if statement?

Reply:
Didn't see that, whoops!